packages/ndla-ui/src/Breadcrumb/BreadcrumbItem.tsx

```diff
@@ -91,13 +91,17 @@ const BreadcrumbItem = forwardRef<any, Props>(
 
     const { to, name, index } = item;
     const isLast = index === totalCount - 1;
-    return (
+    return isLast ? (
+      <StyledListItem ref={liRef} autoCollapse={autoCollapse} aria-current="page">
+        <CollapseContainer autoCollapse={autoCollapse}>
+          <span>{name}</span>
+        </CollapseContainer>
+      </StyledListItem>
+    ) : (
       <StyledListItem ref={liRef} autoCollapse={autoCollapse}>
```

Comment:
Kunne du ikkje like gjerne gjort det slik?
```suggestion
      <StyledListItem ref={liRef} autoCollapse={autoCollapse} aria-current={isLast ? 'page' : undefined}>
```

Ref https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-current
If the attribute is not present, is an empty string, is present with no value, or is set to aria-current="false" it is not exposed to the user.

Reply:
Ja mye bedre!